src/array_api_extra/_lib/_libraries.py

```diff
@@ -0,0 +1,69 @@
+"""Code specifying libraries array-api-extra interacts with."""
+
+from collections.abc import Callable
+from enum import Enum
+from types import ModuleType
+from typing import cast
+
+from ._utils import _compat
+
+__all__ = ["Library"]
+
+
+class Library(Enum):  # numpydoc ignore=PR01,PR02
+    """
+    All array library backends explicitly tested by array-api-extra.
+
+    Parameters
+    ----------
+    value : str
+        String describing the backend.
+    library_name : str
+        Name of the array library of the backend.
+    module_name : str
+        Name of the backend's module.
+    """
+
+    ARRAY_API_STRICT = "array_api_strict", "array_api_strict", "array_api_strict"
```

Comment:
I think I'd rather see an explicit use of `_compat.is_array_api_strict_namespace`.
This would require to add *everything* to `_compat.py`, which is something I wanted to do anyway.